guide-us-english.tex

```diff
@@ -265,7 +265,12 @@ \subsection{Scaling the Product Owner Team}
 
 \section{Scrum Master Cycle - Coordinating the ``How''}
 
-The SM organization (SM's, SoSM's, and EAT) work as a whole to complete the components of the Scrum Master Cycle. The components unique to the Scrum Master Cycle are: \textbf{Continuous Improvement and Impediment Removal, Cross-Team Coordination, and Deployment}.
+The SM organization (SM's, SoSM's, and EAT) work as a whole to complete the components of the Scrum Master Cycle. The components unique to the Scrum Master Cycle are: 
```

Comment:
...I'm kind of neutral on this commit. I don't mind either way. I'd have to see it in final layout.